src/__tests__/exampleData.js

```diff
@@ -125,6 +132,24 @@ export const stream: Stream = makeStream({
   description: 'An example stream.',
 });
 
+export const makeSubscription = (args: { stream?: Stream } = {}): Subscription => {
+  const { stream: streamInner = stream } = args;
+  return deepFreeze({
+    ...streamInner,
+    color: '#123456',
+    in_home_view: true,
+    pin_to_top: false,
+    audible_notifications: false,
+    desktop_notifications: false,
+    email_address: '??? make this value representative before using in a test :)',
```

Comment:
I had a look at `zulip/zulip`'s `zerver/lib/email_mirror_helpers.py:32`, here's how the stream email is generated:

```python
def encode_email_address_helper(name: str, email_token: str, show_sender: bool=False) -> str:
    # Some deployments may not use the email gateway
    if settings.EMAIL_GATEWAY_PATTERN == '':
        return ''

    # Given the fact that we have almost no restrictions on stream names and
    # that what characters are allowed in e-mail addresses is complicated and
    # dependent on context in the address, we opt for a simple scheme:
    # 1. Replace all substrings of non-alphanumeric characters with a single hyphen.
    # 2. Use Django's slugify to convert the resulting name to ascii.
    # 3. If the resulting name is shorter than the name we got in step 1,
    # it means some letters can't be reasonably turned to ascii and have to be dropped,
    # which would mangle the name, so we just skip the name part of the address.
    name = re.sub(r"\W+", '-', name)
    slug_name = slugify(name)
    encoded_name = slug_name if len(slug_name) == len(name) else ''

    # If encoded_name ends up empty, we just skip this part of the address:
    if encoded_name:
        encoded_token = "%s.%s" % (encoded_name, email_token)
    else:
        encoded_token = email_token

    if show_sender:
        encoded_token += ".show-sender"

    return settings.EMAIL_GATEWAY_PATTERN % (encoded_token,)
```

I'm not at all familiar with Python, but it looks like we use a combination of the stream name in ASCII and some server configs to generate the stream name. Perhaps, this might help us in using a representative value here.

Reply:
Yeah -- the easiest way to get a representative value is probably to look at some empirical data, e.g. in the console when debugging the app.

But also it's not a priority to fill in this particular property's value, because we don't have any tests that care about it. I suspect it's something we actually completely don't care about in the app :wink:, and so ideally would just drop and not even store in Redux.